tests/src/Unit/GroupManagerTest.php

```diff
@@ -313,6 +320,15 @@ protected function expectDefaultRoleCreation($entity_type, $bundle) {
   }
 
   /**
+   * Mocked method calls when system under test should attach OG audience field
+   * on the user.
+   */
+  protected function expectGroupCreation() {
```

Comment:
I don't think this needs its own method. You can move the code back to above method.